client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/ILMDocumentationIT.java

```diff
@@ -797,32 +797,11 @@ public void testAddSnapshotLifecyclePolicy() throws Exception {
         // end::slm-put-snapshot-lifecycle-policy-response
         assertTrue(putAcknowledged);
 
-        // tag::slm-put-snapshot-lifecycle-policy-execute-listener
-        ActionListener<AcknowledgedResponse> putListener =
```

Comment:
These are here because they are used as snippets to generate the high level rest client documentation (that's what the surrounding `// tag::etcetc` and `// end::etcetc` are for, so I don't think we should remove these